fedoracommunity/widgets/package/templates/bugs_table_widget.mak

```diff
@@ -23,6 +39,11 @@
                     <option value="${rel['value']}">${rel['label']}</option>
                 % endfor
             </select>
+
+            <span class="pull-right">
+                <label for="version">Search:</label>
+                <input id="bug_search" type="text" name="search" placeholder="Bugzilla..."/>
```

Comment:
Yeah.  it looks kinda okay because there is a ``<label>`` with the word Search just to the left of the ``<input>``... so it reads as "Search Bugzilla..." when you look at it on the page.